ocs_ci/helpers/dr_helpers.py

```diff
@@ -535,3 +536,63 @@ def wait_for_all_resources_deletion(
         pvc_obj.ocp.wait_for_delete(
             resource_name=pvc_obj.name, timeout=timeout, sleep=5
         )
+
+
+def resources_cleanup(namespace):
+    """
+    Cleanup workload and replication resources from managed clusters.
+    Useful for removing resources to avoid further test failures due to leftovers
+
+    Args:
+        namespace(str): the namespace of the workload
+
+    """
+    resources = [
+        constants.POD,
+        constants.VOLUME_REPLICATION_GROUP,
+        constants.VOLUME_REPLICATION,
+        constants.PVC,
+        constants.PV,
+    ]
+    rbd_images = []
+    for cluster in get_non_acm_cluster_config():
+        config.switch_ctx(cluster.MULTICLUSTER["multicluster_index"])
+        for resource in resources:
+            ocp_obj = ocp.OCP(kind=resource, namespace=namespace)
+            for item in ocp_obj.get()["items"]:
+                resource_name = item["metadata"]["name"]
+
+                if resource == constants.PV:
```

Comment:
If these resources are created by workload CRs , I feel pushing this code to dr_workload class would make sense. WDYT ?

Reply:
ack, will move this to dr_workload class